README.md

```diff
@@ -146,65 +159,54 @@ Spin up a new EC2 instance. If using AWS, use the following steps:
 
 🚧 **Coming Soon**
 
-## Managed on-prem deployments
+## Managed deployments
 
 🚧 **Coming Soon**
 
+<!--
 Deploy Datapane on a managed service. We're working on providing some starter template files for Cloudformation setups (ECS + Fargate), Kubernetes, and Helm.
 
-Additionally, we hope to be available in the AWS, Azure, and GCS marketplaces shortly.
-
-## Next Steps
```

Comment:
why is this commented out?

Reply:
oops, looks like my markdown editor borked the comments. will fix.